src/pipeline/eval.py

```diff
@@ -41,23 +43,63 @@ def __batch_process(batch):
         __batch_process, batched=True, batch_size=batch_size
     )
 
+def _calculate_job_distribution(rate_limit_per_minute, num_workers):
```

Comment:
Since both `evaluation` and `synthetic data generation` use Gemini API, I think this helper function should be defined in a separate module/file or `utils.py`.